api/osb/osb_store_plugin.go

```diff
@@ -816,12 +823,38 @@ func (sp *storePlugin) storeInstance(ctx context.Context, storage storage.Reposi
 	return nil
 }
 
+func removeSignatureFromContext(ctx context.Context, rawContext json.RawMessage) (json.RawMessage, error) {
+	var ctxMap map[string]json.RawMessage
+	if err := json.Unmarshal(rawContext, &ctxMap); err != nil {
+		log.C(ctx).Errorf("failed to unmarshal context: %v", err)
+		return nil, err
+	}
+	if _, exists := ctxMap["signature"]; exists {
+		delete(ctxMap, "signature")
+		outputData, err := json.Marshal(ctxMap)
+		if err != nil {
+			log.C(ctx).Errorf("failed to marshal context: %v", err)
+			return nil, err
+		}
+		rawContext = outputData
+	}
+	return rawContext, nil
```

Comment:
-> return sjson.DeleteBytes(rawContext, "signature") ?

Reply:
yes, thank you!! :)